manager/src/test/java/com/redhat/service/smartevents/manager/api/v1/internal/ShardBridgesSyncAPITest.java

```diff
@@ -254,7 +264,57 @@ public void metricsAreProduced() {
                 .body()
                 .asString();
 
-        metricsService.getMetricNames().forEach(metric -> assertThat(metrics).contains(metric));
+        // Not all metrics are recorded by this test. Only the "successful" path in this test.
+        assertThat(metrics).contains(operationTotalCountMetricName);
+        assertThat(metrics).contains(operationTotalSuccessCountMetricName);
+        assertThat(metrics).contains(operationDurationMetricName);
+    }
+
+    @Test
+    @TestSecurity(user = DEFAULT_CUSTOMER_ID)
```

Comment:
@manstis Is this change related?

Reply:
@robpblake it is related; but for the life of me I cannot remember why. It has something to do with needing to split the "metrics" test into "success" and "failure" as not _ALL_ metrics were being (reliably) produced for the single test.